Exiled.API/Features/Player.cs

```diff
@@ -1817,6 +1849,19 @@ public int RemoveItem(Func<Item, bool> predicate, bool destroy = true)
         /// </summary>
         public void ResetStamina() => Stamina = StaminaStat.MaxValue;
 
+        /// <summary>
+        /// Gets a user's SCP preference.
+        /// </summary>
+        /// <param name="roleType">The SCP RoleType.</param>
+        /// <returns>A value from <c>-5</c> to <c>5</c>, representing a player's preference to play as the provided SCP. Will return <see cref="int.MinValue"/> for invalid SCPs.</returns>
+        public int GetScpPreference(RoleTypeId roleType)
+        {
+            if (ScpPreferences.Preferences.TryGetValue(roleType, out int value))
+                return value;
+
+            return int.MinValue;
```

Comment:
Should be 0 i think?

Reply:
actually yeah because 0 is the default, good idea